po/pt_BR.po

```diff
@@ -0,0 +1,706 @@
+
+#: src/install.rs:507
+msgid "Remove make dependencies after install?"
+msgstr "Remover dependências de compilação após instalar?"
+
+#: src/install.rs:518
+msgid "Proceed to review?"
+msgstr "Continuar para análise?"
+
+#: src/install.rs:559
+msgid "The following packages are not compatible with your architecture:"
+msgstr "Os seguintes pacotes não são compatíveis com sua arquitetura:"
+
+#: src/install.rs:573
+msgid "Would you like to try build them anyway?"
+msgstr "Gostaria de compilá-los mesmo assim?"
+
+#: src/install.rs:662
+msgid "packages failed to build: {}"
+msgstr "pacotes"
+
+#: src/install.rs:698
+msgid "could not find .SRINFO for '{}'"
+msgstr "não foi possível encontrar um .SRINFO para '{}'"
+
+#: src/install.rs:762
+msgid "failed to read dir: {}"
+msgstr "erro ao ler diretório: {}"
+
+#: src/install.rs:778
+msgid ""
+"{} is a directory\n"
+"\n"
+msgstr ""
+"{} é um diretório\n"
+"\n"
+
+#: src/install.rs:818
+msgid "failed to open: {}"
+msgstr "erro ao abrir: {}"
+
+#: src/install.rs:830
+msgid "binary file: {}"
+msgstr "arquivo binário: {}"
+
+#: src/install.rs:852
+msgid " nothing new to review"
+msgstr " nada de novo para analisar"
+
+#: src/install.rs:871
+msgid "failed to execute file manager: {}"
+msgstr "erro ao executar gerenciador de arquivos: {}"
+
+#: src/install.rs:874
+msgid "file manager did not execute successfully"
+msgstr "gerenciador de arquivos não foi executado com sucesso"
+
+#: src/install.rs:936
+msgid "duplicate packages: {}"
+msgstr "pacotes duplicados: []"
+
+#: src/install.rs:975
+msgid "could not find all required packages:"
+msgstr "não foi possível encontrar todos os pacotes necessários:"
+
+#: src/install.rs:982
+msgid ""
+"\n"
+"    {:missing} (wanted by: {:stack})"
+msgstr ""
+"\n"
+"    {:missing} (requerido por: {:stack})"
+
+#: src/install.rs:996 src/install.rs:1941
+msgid "{}-{} is up to date -- skipping"
+msgstr "{}-{} está atualizado -- pulando"
+
+#: src/install.rs:1011
+msgid "Calculating conflicts..."
+msgstr "Calculando conflitos..."
+
+#: src/install.rs:1017
+msgid "Calculating inner conflicts..."
+msgstr "Calculando conflitos internos..."
+
+#: src/install.rs:1029
+msgid "Inner conflicts found:"
+msgstr "Conflitos internos encontrados:"
+
+#: src/install.rs:1051
+msgid "Conflicts found:"
+msgstr "Conflitos encontrados:"
+
+#: src/install.rs:1074
+msgid "Conflicting packages will have to be confirmed manually"
+msgstr "Pacotes conflitantes deverão ser confirmados manualmente"
+
+#: src/install.rs:1078
+msgid "can not install conflicting packages with --noconfirm"
+msgstr "não é possível instalar pacotes conflitantes com --noconfirm"
+
+#: src/install.rs:1117
+msgid "Repo"
+msgstr "Repositório"
+
+#: src/install.rs:1124
+msgid "Repo Make"
+msgstr "Pacotes Make no repositório"
+
+#: src/install.rs:1138
+msgid "Aur Make"
+msgstr "Pacotes Make no AUR"
+
+#: src/install.rs:1240
+msgid "fetching devel info..."
+msgstr "buscando informações de compilação..."
```

Comment:
Ideally these will be kept as short as possible. Repository still starts with Repo. Does it make sense in the language to use Repo and Repo Make?

Reply:
Not really, I just followed pacman's own localization, but I'll fix it, no problem

Reply:
Just some parts that really need to be called 'repositórios', because of legibility

Reply:
It's fine if they are. I was just asking, it's not like I know the language.